index.html

```diff
@@ -19,17 +19,28 @@
 
           <div id="menuContentContainer">
             <ul id="menuContent">
-              <li><a href="index.html">Home</a></li>
               <li><a href="disciplines.html">Disciplines</a></li>
               <li><a href="./dropzones.html">Dropzones</a></li>
               <li><a href="#">Where to Start</a></li>
-              <li><a href="#">About</a></li>
               <li><a href="./contact.html">Contact</a></li>
             </ul>
           </div>
         </div>
       </nav>
     </header>
+    <div id="bGImage">
+      <div class="b_g_text">
+        <h1>
+          RAR
+          <div id="spanOne">E</div>
```

Comment:
great job figuring this out but lol you put a div and gave it an id of span hahahaha. Just use a span!! Also "spanOne" is a terrible name and doesnt describe what it is.

Reply:
initially it was just one span property controlling the 2 blacked letters, but when shrinking to phone mobile I couldn't tell them to display the same color. so I had to make them as 2 different declarations.